spec/tags/whatsapp.md

```diff
@@ -34,3 +34,26 @@ When you receive a message from one contact, the sender and recipient is inverte
 * Sender: is the phone number of contact
 
 In API the sender is the field `from` and the receiver is the field `to` of message object.
+
+
+## Type of Messages
+
+* **Template Message**
+
+**What's the template message?**
+
+Message Templates are message formats for common reusable messages a business may want to send. Businesses must use Message Templates for sending notifications to customers.
+
+This allows a business to send just the template identifier along with the appropriate parameters instead of the full message content.
```

Comment:
This seems to suggest that the objective of templates is to reduce the amount of information sent on message posts, but in WhatsApp case, it is the only way to send a message to a user after 24h after his last message or when the user never sent a message. It is actually not recommend to use template message if the user has sent a message in the last 24h.

I think this is easier to explain if messages in general are explained before template messages, which are a special case.